packages/@uppy/react/src/Wrapper.ts

```diff
@@ -0,0 +1,59 @@
+import { createElement as h, Component } from 'react'
+import type { Body, Meta } from '@uppy/utils/lib/UppyFile'
+import type { UIPlugin, Uppy } from '@uppy/core'
+import PropTypes from 'prop-types'
+import { uppy as uppyPropType } from './propTypes.ts'
+
+interface UppyWrapperProps<M extends Meta, B extends Body> {
+  uppy: Uppy<M, B>
+  plugin: string
+}
+
+class UppyWrapper<M extends Meta, B extends Body> extends Component<
```

Comment:
Is this still used? I'm not sure what it does

Reply:
Yes it is exported: https://github.com/transloadit/uppy/blob/aa2c53254e0a7f096f7816331af9f7e834a3efb7/packages/@uppy/react/src/Wrapper.ts#L59